include/drivers/yen/ksp_driver.h

```diff
@@ -4,6 +4,9 @@ File: ksp_driver.h
 Copyright (c) 2015 Celia Virginia Vergara Castillo
 Mail: vicky_vergara@hotmail.com
 
+Copyright (c) 2023 Aniket Agarwal
+Mail: aniketgarg187@gmail.com
```

Comment:
Use ` at ` instead of `@` one of our goals for v4 is to not have clickable mails to avoid unwanted mails from robots

Reply:
vicky at erosion.dev